### PR TITLE
fix(exec): Better Support for Containerized Executor

### PR DESCRIPTION
**Pull request by RPate97**
# Purpose
- Fixes some minor issues with using Alchemy instead of Infura for the executor
- Increases the loop interval to reduce provider usage to remain within Alchemy free tier for the moment
- Updates plugins hardhat.config.ts to be consistent about using Alchemy instead of Infura
- Refactors Etherscan API key to use a single environment variable so that the env variable setup is consistent across networks

### CODE REVIEW

File: packages/executor/src/index.ts [hunk 1]
```diff
@@ -175,7 +172,7 @@ export class ChugSplashExecutor extends BaseServiceV2<Options, Metrics, State> {
     // Get approval events in blocks after the stored block number
     const newApprovalEvents = await registry.queryFilter(
       registry.filters.EventAnnounced('ChugSplashBundleApproved'),
-      this.state.lastBlockNumber + 1,
+      this.state.lastBlockNumber,
```
Comment (RPate97) [inline packages/executor/src/index.ts:175]: Alchemy handles event queries slightly differently.

File: packages/executor/src/utils/constants.ts [hunk 2]
```diff
@@ -12,13 +12,14 @@ export const customChains = [
...
     },
   },
+  {
+    network: 'arbitrum-goerli',
+    chainId: 421613,
+    urls: {
+      apiURL: 'https://api-goerli.arbiscan.io/api',
+      browserURL: 'https://goerli.arbiscan.io/',
+    },
+  },
```
Comment (RPate97) [inline packages/executor/src/utils/constants.ts:22]: I tested this on Arbitrum, b/c live network deployments still were not working for me.

File: packages/executor/src/utils/constants.ts [hunk 3]
```diff
@@ -12,13 +12,14 @@ export const customChains = [
...
 
-export const etherscanApiKey = {
-  optimisticGoerli: process.env.OPT_ETHERSCAN_API_KEY
-    ? process.env.OPT_ETHERSCAN_API_KEY
-    : '',
-  goerli: process.env.ETH_ETHERSCAN_API_KEY
-    ? process.env.ETH_ETHERSCAN_API_KEY
-    : '',
-}
+export const etherscanApiKey = process.env.ETHERSCAN_API_KEY
```
Comment (RPate97) [inline packages/executor/src/utils/constants.ts:25]: Using a consistent set of env variables makes deploying multiple containers with Terraform easier.

**Comment by sam-goldman** [comment]
For posterity: the BaseServiceV2 loop was changed from 1 second to 5 seconds to reduce AWS costs.

**Review by sam-goldman — APPROVED** [review]

**Pull request merged**